fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/version/GetLatestVersionDAO.java

```diff
@@ -50,7 +50,7 @@ public GetLatestVersionDAO(String adminSchemaName, String schemaName) {
         final String cols = DataDefinitionUtil.join(SchemaConstants.SCHEMA_NAME, SchemaConstants.OBJECT_TYPE, SchemaConstants.OBJECT_NAME);
         final String sql = "SELECT " + cols
                 + ", max(" + SchemaConstants.VERSION + ") FROM " + tbl
-                + " WHERE " + SchemaConstants.SCHEMA_NAME + " IN (?, ?) "
+                + " WHERE UCASE(" + SchemaConstants.SCHEMA_NAME + ") IN (UCASE(?), UCASE(?)) "
```

Comment:
I wrapped this in ucase for backwards compatiblity

Reply:
swapped to  UPPER

Reply:
I don't think you should UCASE(?) the args like that...because it suggests you don't know what case the args are. We should be making sure the args are upper-case before that (because it's probably important for equals and HashMap lookups in other places). The first place I'd force case is when we parse the CLI argument in Main.

Reply:
OK -  I sort of agree, However there is an inconsistency across versions I'd like too account for,  also schema names in postgres tend  to be lowercase versus uppercase  in db2.

Reply:
I was thinking normalizing casing here was critical for backwards compatibility.  If there is an existing version history service value thats been saved as lowercase, we need the new uppercase schema name to match it, right?

Reply:
Correct, I do that when we process the result sets down further on in